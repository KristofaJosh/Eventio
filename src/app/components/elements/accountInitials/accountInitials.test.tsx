import React from "react";
import renderer from "react-test-renderer";
import AccountInitials from "./index";

it("renders correctly", () => {
    const tree = renderer.create(<AccountInitials name="James brown" />).toJSON();
    expect(tree).toMatchSnapshot();
});
