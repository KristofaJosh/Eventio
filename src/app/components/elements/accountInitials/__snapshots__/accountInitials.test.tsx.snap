// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="initials  "
>
  <p>
    J b
  </p>
</div>
`;
